docs/WinML_principles.md

```diff
@@ -0,0 +1,37 @@
+# Contributing to Windows ML
+
+Window Machine Learning is a high-performance, reliable API for deploying hardware-accelerated ML inferences on Windows devices. Please visit the [Windows ML documentation](https://docs.microsoft.com/en-us/windows/ai/windows-ml/) to learn more about Windows Machine Learning. 
+
+
+## Windows ML Base Principles
+
+**We design and optimize for all Windows devices.**
+  
+  Our goal is to provide developers with a platform that enables new experiences that runs well on all Windows devices. Our design drives innovation in the DirectX ecosystem through DirectML and gives developers the confidence that their applications will work for all Windows customers.
+
+**We maintain and curate the Windows ML APIs.**
+    
+  The API is designed to ensure consistency of developer’s experience across the Windows platform. We provide long-term servicing and support, and we are committed to ensuring application’s compatibility as we evolve the API.
+
+**Windows ML is a core component of Windows.**
+    
+  The Windows ML source code is packaged and distributed with each new release of Windows. To provide consumers with high-quality products, Microsoft is responsible for distributing Windows ML and related binaries as part of Windows or standalone distributable packages.
+
+## Keeping the bar high
+
+We support community contribution to Windows ML based on our principles to enhance user experience on Windows. While we encourage everyone to contribute to Windows ML, the Windows ML team will keep the bar for the acceptance high. Help us to do so by following the recommendations bellow.
+  * The feature request is aligned with the Windows ML base principles. 
+  * The feature request addresses the real existing or future user problems.
+  * The feature request focuses on the enhancement of user experience and benefits a meaningful number of users. 
+  
+## When there is no need in feature request
+
+  * When the required change is fixing a bug in Windows ML APIs. In this case, please fill the bug/performance issue request. 
```

Comment:
this applies when the bug is in the implementation of the APIs. If there is a bug in the API itself such that a change is needed to the method signature or equivalent, then we would need to do an official review.